src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -146,6 +179,22 @@ public void deleteEntry_entryIsSelectedAndSecondEntryInFilteredEntryList_firstEn
         modelManager.setSelectedEntry(BOB);
         modelManager.deleteListEntry(BOB);
         assertEquals(ALICE, modelManager.getSelectedEntry());
+
```

Comment:
Again these test abuses the fact that we don't do duplicate checks at `add*Entry()` in `Model`. Maybe we split them up?

Reply:
Done.